packages/react-native-sdk/src/hooks/useAndroidKeepCallAliveEffect.ts

```diff
@@ -23,6 +23,13 @@ async function startForegroundService() {
   const foregroundServiceConfig = StreamVideoRN.getConfig().foregroundService;
   const { title, body } = foregroundServiceConfig.android.notificationTexts;
   const channelId = foregroundServiceConfig.android.channel.id;
+
+  // request for notification permission and then start the foreground service
+  const settings = await notifee.getNotificationSettings();
+  if (settings.authorizationStatus !== AuthorizationStatus.AUTHORIZED) {
+    console.warn('Notification permission not granted');
+    return;
+  }
```

Comment:
This warn/check is unnecessary.. as notifee will be a noop if notification permission is not granted..

If we really want to warn.. lets make the warning descriptive.. this doesnt help the integrator to understand why the warning comes in my view

Reply:
what should we add to make it more desc. ?